src/utils.jl

```diff
@@ -266,3 +266,11 @@ function random_control_problems(n::Integer, m::Integer; ivp=nothing, period=0.1
 
     return problems
 end
+
+include("nnops.jl")
+# relative size between the set-based output and the (CH of) sampled output
+function relative_size_nnsol(X0, nsamples, solver, controller)
```

Comment:
i think we can call this `relative_size` and change the signature to `relative_size(X0, nsampler, controller, solver=Ai2z())`.

Reply:
maybe add an assertion `@assert dim(X0) == 1`